### PR TITLE
Add auto-mode-alist entry

### PR DESCRIPTION
**Pull request by fbergroth**

### CODE REVIEW

File: README.org [hunk 1]
```diff
@@ -17,8 +17,9 @@ faces, making sure that they fit in with any theme.
 Features and settings outside font-locking are not touched and will be inherited from ~conf-space-mode~.
 
 ** How to use
-Place the line ~# -*- mode: i3wm-config -*-~ at the top of your i3wm config file (as seen in the first screenshot) and
-Emacs will automatically load and activate ~i3wm-config-mode~ when the file is opened.
+This mode will activate for files who's path ends with ~.../i3/config~. If you use a different name, you can activate
```
Comment (Alexander-Miller) [inline README.org]: whose

File: i3wm-config-mode.el [hunk 2]
```diff
@@ -348,6 +348,9 @@ the 'x' in 'for_window [x=y]'.")
 (define-derived-mode i3wm-config-mode conf-space-mode "i3wm Config"
   (font-lock-add-keywords nil i3wm-config-font-lock-keywords 'set))
 
+;;;###autoload
+(add-to-list 'auto-mode-alist '("/i3/config\\'" . i3wm-config-mode))
```
Comment (Alexander-Miller) [inline i3wm-config-mode.el:352]: Neat, I didn't know auto-mode-alist can match part of the file path as well.

Also squash the commits, and we're good to go.

**Review by Alexander-Miller — CHANGES_REQUESTED** [review]

File: README.org (hunk 1 above)
Comment (fbergroth) [reply README.org]: Thanks!

File: i3wm-config-mode.el (hunk 2 above)
Comment (fbergroth) [reply i3wm-config-mode.el:352]: Done!

**Review by Alexander-Miller — APPROVED** [review]

**Comment by Alexander-Miller** [comment]
:+1:

**Comment by erhandsome** [comment]
i3wm also has config path ~/.i3/config, this location won't work, maybe `(add-to-list 'auto-mode-alist '("i3/config\\'" . i3wm-config-mode))` ?

**Comment by Alexander-Miller** [comment]
@erhandsome Good catch. I've pushed a patch to fix it now.

**Pull request merged**